audit/projects/kubernetes-public/services/enabled.txt

```diff
@@ -15,7 +16,9 @@ logging.googleapis.com               Cloud Logging API
 monitoring.googleapis.com            Cloud Monitoring API
 oslogin.googleapis.com               Cloud OS Login API
 pubsub.googleapis.com                Cloud Pub/Sub API
+secretmanager.googleapis.com         Secret Manager API
```

Comment:
I think this may be the new part of the audit/logging issues.
I suspect folks who have run the audit-gcp.sh script since adding secrets have had access to secret manager.